### ISSUE TITLE
Stable release

### ISSUE THREAD

**Issue by simPod** [S1lentium/IPTools#27]
Hello, there have not been a stable release for a while and we have to use `dev-master@dev` as required version which is not very fancy.

Do you plan to tag a version in a near future?

Thanks!

**Comment by simPod** [comment]
bump @S1lentium

**Comment by S1lentium** [comment]
There are 3 stable releases for v1:
https://github.com/S1lentium/IPTools/releases

**Comment by simPod** [comment]
Yes, but last was in 2016 https://github.com/S1lentium/IPTools/tree/v1.1.0

![image](https://user-images.githubusercontent.com/327717/45253018-390a2a80-b360-11e8-8967-b2ecb1e37c90.png)
![image](https://user-images.githubusercontent.com/327717/45253019-41626580-b360-11e8-8c10-17e922eefd7f.png)

What about next one after 2 years? ;)
